ibm/service/vpc/data_source_ibm_is_instance_profile.go

```diff
@@ -461,6 +461,31 @@ func DataSourceIBMISInstanceProfile() *schema.Resource {
 					},
 				},
 			},
+			"network_interface_count": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"max": {
+							Type:        schema.TypeInt,
+							Computed:    true,
+							Optional:    true,
```

Comment:
here as well remove optional in all 3 places

Reply:
done